src/v2/views/MonsterCollectionOverlay/index.tsx

```diff
@@ -87,6 +92,11 @@ function MonsterCollectionOverlay({ isOpen, onClose }: OverlayProps) {
             collectionSheet={collection.stateQuery.monsterCollectionSheet}
             onActionTxId={(txId) => {
               setLoading(true);
+              ipcRenderer.send("mixpanel-track-event", "Staking/Migration", {
+                txId,
+                tip: tip.nodeStatus.tip.index,
+                amount: collection.stateQuery.monsterCollectionState?.level,
```

Comment:
If the `amount` was intended to represent its deposit, it seems not to represent it correctly. In my thought, the `level` naming is enough to represent the amount. The `level`s can be converted to `amount`.

Maybe it needs `amount` when the Mixpanel user wants to summarize them for the total migrated amount. But then, it needs to fetch `query.stateQuery.monsterCollectionState.address`' deposit from `query.goldBalance(address)`.

Reply:
I'm sorry that I didn't see the PR description 😢 The above comment is for only the naming.

Reply:
Oh right, I tried to convert the level to the amount by looking it up on the sheet but I haven't really done them. I'll rename it to level.